packages/account/src/Components/poa/common-mistakes/common-mistake-examples.tsx

```diff
@@ -4,7 +4,7 @@ import { isMobile } from '@deriv/shared';
 import { Localize } from '@deriv/translations';
 import { useStore } from '@deriv/stores';
 import { getExampleImagesConfig } from '../../../Configs/poa-common-mistake-examples-config';
-import IcErrorBadge from '../../../Assets/ic-error-badge.svg';
+import { LegacyLossIcon } from '@deriv/quill-icons';
```

Comment:
Please remove the svgs from the Assets folder

Reply:
done